pyaedt/edb_core/layout.py

```diff
@@ -12,11 +12,11 @@
     from System import Double, Array
     from System.Collections.Generic import List
 except ImportError:
-    warnings.warn('This module requires pythonnet.')
+    warnings.warn('This module requires Python.NET.')
```

Comment:
```suggestion
    warnings.warn('This module requires `pythonnet`.')
```
This is the name of a python package. So maybe it should be left as is with the name wrapped in tick marks?

Reply:
I realize now, this is not a docstring so the tick marks have no affect. Is there a better way to denote `pythonnet` as the name of a packge?